R/Samples-methods.R

```diff
@@ -50,6 +50,8 @@ setMethod(
 ## plots with "ggmcmc" package
 ## --------------------------------------------------
 
```

Comment:
please add a comment to explain the necessity of this, otherwise we might remove this in the future.